package.json

```diff
@@ -129,6 +130,7 @@
     "json-rpc-engine": "^5.3.0",
     "json-rpc-middleware-stream": "^2.1.1",
     "jsonschema": "^1.2.4",
+    "lavamoat-core": "^6.1.0",
```

Comment:
I think this should be a `devDependency`, since it's only being used in the build system at the moment.

Reply:
Sounds good!